contracts/test/TestWarmColdAccount.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.12;
+/* solhint-disable no-inline-assembly */
+
+import "../interfaces/IAggregatedAccount.sol";
+import "../interfaces/IEntryPoint.sol";
+
+
+contract TestWarmColdAccount is IAggregatedAccount {
+    IEntryPoint private ep;
+    uint public state = 1;
+    constructor(IEntryPoint _ep) payable {
+        ep = _ep;
+    }
+
+    function validateUserOp(UserOperation calldata userOp, bytes32, address, uint256 missingAccountFunds)
+    external override returns (uint256 sigTimeRange) {
+        ep.depositTo{value : missingAccountFunds}(address(this));
+        if (userOp.nonce == 1) {
+            this.touchStorage{gas: 1000}();
```

Comment:
missing comments what this "1000" means. These tests should be educational... also, I wonder how to refer to the failed EntryPoint. maybe add to the comment "test fails on EP from commit 0x123456". 
To verify test test one should check out that commit, and separately get the tests (which exist in the next commit). not that trivial, but at least possible.

Reply:
Added comments